From d574809aedb4aca253e18b282117d3141caf58e9 Mon Sep 17 00:00:00 2001
From: Bin Yang <yangbin@rock-chips.com>
Date: Tue, 8 May 2018 17:14:17 +0800
Subject: [PATCH] arm64: dts: rockchip: modify pmu_io_domains for
 rk3399-tve1030g

pmu_io_domains voltage is 1.8V for rk3399-tve1030g board

Change-Id: Iffba406ca1e88f346fc9e436b7b4ea4878b19e92
Signed-off-by: Bin Yang <yangbin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dts b/arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dts
index 9b3abd64aa8c..2a2dbb33efe9 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dts
@@ -791,7 +791,7 @@
 
 &pmu_io_domains {
 	status = "okay";
-	pmu1830-supply = <&vcc_3v0>;
+	pmu1830-supply = <&vcc_1v8>;
 };
 
 &pwm0 {
-- 
2.35.3

